modules/cli/src/main/scala/scala/cli/launcher/LauncherOptions.scala

```diff
@@ -0,0 +1,21 @@
+package scala.cli.launcher
+
+import caseapp._
+import upickle.default.{ReadWriter, macroRW}
+
+@HelpMessage("Set the ScalaCLI version")
+final case class LauncherOptions(
+  @Group("Launcher")
+  scalaCliVersion: Option[String] = None,
```

Comment:
Maybe a shorter option would be better, like
```suggestion
  cliVersion: Option[String] = None,
```